### ISSUE TITLE
Unknown TZ UTC error Import CSV using RStudio

### ISSUE THREAD

**Issue by OEHtkoen** [OEHtkoen/TerrysStuff#1]
Dear Helpers

(This problem may be computer specific - the problem listed below occurs on my Lenovo R400 ThinkPad laptop, but does not occur on my Lenovo ThinkCentre M Series desktop)

RStudio V1.0.143

> sessionInfo()
R version 3.4.0 (2017-04-21)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 7 x64 (build 7601) Service Pack 1
Matrix products: default
locale:
[1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
[3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_3.4.0 tools_3.4.0   

When using RStudio Import Dataset dropdown to import a CSV, I get the error "Is this a valid CSV file? Unknown TZ UTC". The import works from this menu if I change the default Locale / Configure / Time Zone from UTC to '' (ie blank).  Any ideas as to why I have this stumble?

From the R command prompt, I can see these settings
> base::Sys.timezone()
[1] "Australia/Sydney"
and > base::OlsonNames() lists 572 names, 19 of which are "Australia/somewhere".

Also at the command prompt, executing these commands
> Sys.setenv(TZ = Sys.timezone())
> options(readr.default_locale = readr::locale(tz = Sys.timezone()))

does nothing to solve the issue with RStudio's Import, however, performing the input manually by typing this next line DOES lead to success.
> pilligatrfreq <- read_csv("K:/MichaelMurphy/Pilliga/pilligatrfreq.csv")

To summarise - success with either RStudio by deleting the Time Zone 'UTC' for a blank entry, the import command becomes

pilligatrfreq <- read_csv("K:/MichaelMurphy/Pilliga/pilligatrfreq.csv",  locale = locale(tz = ""))

or at the R prompt, run the 2 commands above followed by the read_csv command.

pilligatrfreq <- read_csv("K:/MichaelMurphy/Pilliga/pilligatrfreq.csv")

Many thanks for any insights, Terry
